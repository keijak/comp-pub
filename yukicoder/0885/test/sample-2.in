10
3 11 6 2 34 21 44 8 0 2
10
50 34 42 36 26 8 16 13 24 4
