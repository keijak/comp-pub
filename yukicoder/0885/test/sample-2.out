131
63
63
63
63
23
23
23
23
15
